### DISCUSSION TITLE
Password protecting access to Frigate

### DISCUSSION THREAD

**Discussion by outofsight** [blakeblackshear/frigate#3112] [Ask A Question]
Just a simple question. How to preotect access to frigate page, for example http://nvr:5000/, to prevent access from users in the same LAN?

**Comment by NickM-27** [comment]
There's currently not a first party solution for this. There are some ways this could probably be done, for example closing off the standard 5000 port and using a reverse proxy like nginx with HTTPS auth enabled.
